### PR TITLE
Update openai

### PR DESCRIPTION
**Pull request by SakuraSakuraSakuraChan**
Update openai related domain

### CODE REVIEW

File: data/openai [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+ai.com
+
+# Api
+auth0.com
+sentry.io
+unpkg.com
+
+# CDN
+full:openaiapi-site.azureedge.net
+full:challenges.cloudflare.com
```
Comment (rootmelo92118) [inline data/openai:12]: We need more evidence to prove lines `6` `7` `8` and `12` belong to `openai`.

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: data/openai (hunk 1 above)
Comment (SakuraSakuraSakuraChan) [reply data/openai:12]: They don't, openai is using these services

Comment (rootmelo92118) [reply data/openai:12]: They should not include in this file if they are not belonging to `openai` or are only used on `openai`.

**Comment by IceCodeNew** [comment]
@rootmelo92118 please notify me when you think this PR is ready to go.

**Comment by rootmelo92118** [comment]
@IceCodeNew Done.

File: data/openai [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # Main domain
 openai.com
+ai.com
```
Comment (IceCodeNew) [inline data/openai:3]:
```suggestion
ai.com
openai.com
```

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**